src/pages/hayoungGam/Detail/DrinkReview/DrinkReview.js

```diff
@@ -0,0 +1,98 @@
+import { Component } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import DrinkReviewComment from '../DrinkReviewComment/DrinkReviewComment';
+import './DrinkReview.scss';
+
+class DrinkReview extends Component {
+  constructor() {
+    super();
+    this.state = {
+      reviews: [],
+    };
+  }
+
+  removeReviews = comment => {
+    const commentId = comment.target.parentNode.id;
+    this.setState({
+      reviews: this.state.reviews.filter(review => review.id !== commentId),
+    });
+  };
+
+  saveInputValue = event => {
+    event.preventDefault();
+    const {
+      target: {
+        reviewerId: { value: userId },
+      },
+    } = event;
```

Comment:
```suggestion
    const userId = event.target.reviewerId.value.userId;
```
이게 더 가독성이 좋을 것 같아요 :)

Reply:
지적 감사합니다! 수정했습니다!

https://github.com/wecode-bootcamp-korea/1-React-Webucks-1/pull/1/commits/79f24adc45eeb777d2e3baa89806da16993c2f54